libr/core/cmd_print.inc.c

```diff
@@ -3634,15 +3606,15 @@ static bool cmd_print_ph(RCore *core, const char *input) {
 		return true;
 	}
 	if (!i0 || i0 == 'l' || i0 == 'L') {
-		algolist (core, 1);
+		RCrypto *cry = r_crypto_new ();
+		r_crypto_list (cry, NULL, 'q', R_CRYPTO_TYPE_HASHER);
+		r_crypto_free (cry);
 		return true;
 	}
 	if (i0 == 'j') {
-		algolist (core, 'j');
-		return true;
-	}
-	if (i0 == '=') {
-		algolist (core, 0);
+		RCrypto *cry = r_crypto_new ();
```

Comment:
i think we should have a crypto instance inside RCore, and then use get_session() to associate the requested plugins to do the cyrpto operations over it.